recipes/bioconductor-optilcms/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "OptiLCMS" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: 'bioconductor-{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: 'https://github.com/xia-lab/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz'
+  sha256: a310974ea2aae951e078654c24bd331aa662d98191c75fcb82d9b13ce2f2b084
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  script: 'R CMD INSTALL --build .'
+  run_exports:
+    - '{{ pin_subpackage("bioconductor-optilcms", max_pin="x.x.x") }}'
```

Comment:
Are you sure about `x.x.x` ?
See the Pull Request template for an explanation of the meanings - https://github.com/bioconda/bioconda-recipes/blob/master/.github/PULL_REQUEST_TEMPLATE.md#instructions-for-avoiding-api-abi-and-cli-breakage-issues

Reply:
Change to `"x"`